docker/README.md

```diff
@@ -1,8 +1,8 @@
 # SyTest Docker Images
 
 These Dockerfiles create containers for running SyTest in various
-configurations. SyTest is not included in these images, but its dependencies
-are.
+configurations. SyTest is not [included in these images by default](#selecting-a-sytest-checkout),
+but its dependencies are.
```

Comment:
it's not actually possible to include Sytest itself in the Docker images. How about:


```suggestion
configurations. SyTest is not included in these images (see 
[Selecting a checkout of SyTest](#selecting-a-sytest-checkout)),
but its dependencies are.
```